backend/src/appointment/routes/caldav.py

```diff
@@ -37,6 +37,13 @@ def caldav_autodiscover_auth(
 
     dns_lookup_cache_key = f'dns:{utils.encrypt(connection.url)}'
 
+    # Check for an attempt to use Google CalDAV API
+    # which we don't support because we use their API directly
+    basename = urlparse(connection.url).netloc
+    google_domains = ['googleusercontent.com', 'google.com', 'gmail.com']
+    if (any([(g in basename) for g in google_domains])):
+        raise GoogleCaldavNotSupported()
+
```

Comment:
Suggestion: It might be good to add an integration test for this scenario, to https://github.com/thunderbird/appointment/blob/main/backend/test/integration/test_calendar.py.

Reply:
Always forgetting about the tests 😂  Thanks, I'll do that!

Reply:
I added a corresponding test now. Thanks again for the reminder.